packages/clusview/samplers/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Parameters?
```

Comment:
Samplers generate and iterate over predefined ranges. Yes, they can do this over simple one-dimensional parameters, but this can also happen over multiple fields of object instances. Essentially, over anything that can have a range of values.